### PR TITLE
One pl_pl fix

### PR DESCRIPTION
**Pull request by jogurciQ**

### CODE REVIEW

File: src/main/resources/assets/upgrade_aquatic/lang/pl_pl.json [hunk 1]
```diff
@@ -537,8 +537,6 @@
   "tooltip.upgrade_aquatic.pink_jelly_torch": "różową galaretkową pochodnię",
   "tooltip.upgrade_aquatic.purple_jelly_torch": "fioletową galaretkową pochodnię",
   "tooltip.upgrade_aquatic.white_jelly_torch": "białą galaretkową pochodnię",
-  
-  "tooltip.upgrade_aquatic.unobtainable": "Nie do zdobycia",
```
Comment (SmellyModder) [inline src/main/resources/assets/upgrade_aquatic/lang/pl_pl.json:541]: Why should this get removed?

**Review by SmellyModder — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/upgrade_aquatic/lang/pl_pl.json (hunk 1 above)
Comment (jogurciQ) [reply src/main/resources/assets/upgrade_aquatic/lang/pl_pl.json:541]: Oh I didn't notice it getting removed

**Comment by jogurciQ** [comment]
Fixed it

File: src/main/resources/assets/upgrade_aquatic/lang/pl_pl.json (hunk 1 above)
Comment (jogurciQ) [reply src/main/resources/assets/upgrade_aquatic/lang/pl_pl.json]: Fixed it

**Review by SmellyModder — APPROVED** [review]

**Pull request merged**